phpBB/styles/prosilver/template/mcp_post.html

```diff
@@ -168,6 +168,10 @@
 			</form>
 		<!-- ENDIF -->
 
+		<!-- IF S_MCP_POST_ADDITIONAL_OPTS -->
```

Comment:
The problem is that this is within another IF block, where the conditions are limited (see above, line 147), so the template variable is needed anyway.  Or else, the broader block should be broken, but that will create issues with the divs around it.